app/views/upload/register/upload-register.njk

```diff
@@ -0,0 +1,41 @@
+{% extends '_layout.njk' %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+
+{% set pageHeading = "Upload register" %}
+
+{% block content %}
+  <form method="post" novalidate="novalidate" enctype="multipart/form-data">
+    {% block formContent %}
+      <div class="govuk-body">
```

Comment:
Do we need to add some explanatory text that this process will look for new records and only add new records to the DB?

Reply:
It's more of a proof of concept for now. We will need full requirements, guidance text can be added later once we have a better idea.